infra/lib/stack.ts

```diff
@@ -0,0 +1,191 @@
+/**
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as cdk from "aws-cdk-lib";
+import { CfnOutput } from "aws-cdk-lib";
+import * as wafv2 from "aws-cdk-lib/aws-wafv2";
+import { Construct } from "constructs";
+import { AnalyticsIntegrationTestStack } from "./analytics/stack";
+import {
+  AuthIntegrationTestStack,
+  AuthIntegrationTestStackEnvironmentProps
+} from "./auth/stack";
+import { AmplifyCategory, IntegrationTestStack } from "./common";
+import {
+  StorageAccessLevel,
+  StorageIntegrationTestStack
+} from "./storage/stack";
+
+export class AmplifyFlutterIntegStack extends cdk.Stack {
+  constructor(scope: Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    // Allow overriding which categories to deploy
+    const categoryParam = this.node.tryGetContext("category");
+    const categories: AmplifyCategory[] = categoryParam
+      ? categoryParam.split(",")
+      : Object.values(AmplifyCategory);
+
+    console.log(`Deploying stacks for categories: ${categories}`);
```

Comment:
I ran `./tool/provision_integration_test_resources.sh analytics,auth,storage` and this line output: `Deploying stacks for categories: analytics,api,auth,dataStore,storage`

Reply:
Looks like it prints during bootstrap for some reason... if you look further down when it deploys, it prints correctly